apps/docs/src/guide/quickstart/index.md

```diff
@@ -6,10 +6,46 @@ You can quickly bootstrap a full-stack Fuel project locally with the following c
 
 ```sh [pnpm]
 pnpm create fuels
+
+# or, if you want to pass in your preferences directly from the command line:
+pnpm create fuels [project-name] [options]
+
+# eg. pnpm create fuels my-fuel-project --pnpm --contract --predicate
+# Note: project-name and all other options are optional
+Options:
+  -V, --version    output the version number
+  -c, --contract   Include contract program
+  -p, --predicate  Include predicate program
+  -s, --script     Include script program
+  --pnpm           Use pnpm as the package manager
+  --npm            Use npm as the package manager
+  --cs             Include contract and script programs
+  --cp             Include contract and predicate programs
+  --ps             Include predicate and script programs
+  --cps            Include all programs
```

Comment:
I suspect you want to use single dashes when combining options. 🤔 

```suggestion
  -cs             Include contract and script programs
  -cp             Include contract and predicate programs
  -ps             Include predicate and script programs
  -cps            Include all programs
```

Reply:
I'd favor a one-liner here:
```
-cs, -cp, -sp, -cps    Shorthand to include combination of contract, script and predicate programs
```